source/documentation/administration_guide/topics/Virtual_Machine_Pool_Host_settings_explained.adoc

```diff
@@ -57,10 +57,19 @@ If the network connection breaks prior to the completion of the post-copy proces
 |*Enable migration encryption*
 a| Allows the virtual machine to be encrypted during migration.
 
-* *Cluster default (Don't encrypt)*
+* *Cluster default*
 * *Encrypt*
 * *Don't encrypt*
 |
+|*Parallel Migrations* a|Allows you to specify whether and how many parallel migration connections to use.
+
+* *Disabled*: The virtual machine is migrated using a single, non-parallel connection.
+* *Auto*: The number of parallel connections is automatically determined. This settings might automatically disable parallel connections.
+* *Auto Parallel*: The number of parallel connections is automatically determined.
+* *Custom*: Allows you to specify the preferred number of parallel connections, the actual number may be lower.
```

Comment:
Is "Cluster default" option intentionally omitted here ...

Reply:
@mz-pdm, I don't see a Cluster default under the Parallel Migrations option.
![Screenshot from 2022-04-07 10-38-20](https://user-images.githubusercontent.com/76975926/162254435-20edb978-b27b-40d4-9c22-2083c9a9a12f.png)

Reply:
I see, it's actually there on the VM Host setting. I'll add it to the docs where appropriate.